simplicity_sdk/platform/security/sl_component/sli_psec_osal/inc/sli_psec_osal.h

```diff
@@ -42,7 +42,10 @@
   #include "sl_component_catalog.h"
 #endif
 
-#if defined(SL_CATALOG_MICRIUMOS_KERNEL_PRESENT) || defined(SL_CATALOG_FREERTOS_KERNEL_PRESENT)
+#if defined(__ZEPHYR__)
+  #include "sli_psec_osal_zephyr.h"
+  #define SLI_PSEC_THREADING
+#elif defined(SL_CATALOG_MICRIUMOS_KERNEL_PRESENT) || defined(SL_CATALOG_FREERTOS_KERNEL_PRESENT)
```

Comment:
I suggest have a commit with raw file as they appear in the upstream and another commit with the changes we apply to the upstream (I thought we already did that).

(Maybe the scripts could be in a third commit)